articles/cdr-scale-barriers/components/big-quote.js

```diff
@@ -0,0 +1,64 @@
+import { Box } from 'theme-ui'
+import { Row, Column } from '@carbonplan/components'
+
+const COLORS = {
+  provider: 'teal',
+  buyer: 'green',
+  broker: 'orange',
+  'system actor': 'yellow',
+}
+
+const BigQuote = ({ category, quote }) => {
+  return (
+    <Box
+      sx={{
+        borderTop: ({ colors }) => `solid 1px ${colors.muted}`,
+        my: [6, 6, 6, 7],
+        pt: [4, 4, 4, 5],
+        pb: [2, 2, 2, 3],
+      }}
+    >
+      <Box
+        sx={{
+          position: 'relative',
+          fontSize: [2, 2, 2, 3],
+        }}
+      >
+        <Box
+          sx={{
+            position: 'absolute',
+            top: ['-36px', '-36px', '-36px', '-45px'],
+            right: '0px',
+            textAlign: 'right',
+            fontFamily: 'heading',
+            letterSpacing: 'smallcaps',
+            textTransform: 'uppercase',
+            bg: 'background',
+            pl: [4],
+            color: COLORS[category],
+          }}
+        >
+          {category}
+        </Box>
+      </Box>
+      <Row columns={6}>
+        <Column start={1} width={5}>
```

Comment:
At the largest breakpoint, the quote content feels a _little_ wide. We could try narrowing to four columns at that breakpoint. Don't feel particularly strongly about this though!

now:
<img width="1475" alt="Screen Shot 2022-07-12 at 8 32 47 AM" src="https://user-images.githubusercontent.com/12436887/178528479-da83172e-7bcf-4718-b261-e070c62d93e7.png">
alternative:
<img width="1499" alt="Screen Shot 2022-07-12 at 8 32 32 AM" src="https://user-images.githubusercontent.com/12436887/178528463-c2b5c9b3-60d7-4250-ba0a-74e75ad1c399.png">